packages/admin-panel/src/editor/Editor.js

```diff
@@ -7,18 +7,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { InputField } from '../widgets';
 import { checkVisibilityCriteriaAreMet, labelToId } from '../utilities';
+import { createBase64Image } from '../utilities/createBase64Image';
+import styled from 'styled-components';
+
+const EditorWrapper = styled.div`
+  .file_upload_label {
+    font-size: 0.9375rem;
+    text-transform: initial;
+    color: ${props => props.theme.palette.text.secondary};
+  }
+`;
 
 export const Editor = ({ fields, recordData, onEditField }) => {
-  const onInputChange = (inputKey, inputValue, editConfig = {}) => {
-    const { setFieldsOnChange } = editConfig;
+  const onInputChange = async (inputKey, inputValue, editConfig = {}) => {
```

Comment:
I don't love the switching to `async` here. It probably would be fine, but any idea if we could re-work the `createBase64Image` function to be synchronous using `FileReaderSync`? It seems broadly supported across browers, though it does require being in worker threads... No biggie but just wanted to suggest it